jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/WebAppDefaultServletCacheTest.java

```diff
@@ -34,7 +34,7 @@ public class WebAppDefaultServletCacheTest
     private Path resourcePath;
 
     @BeforeEach
-    protected void setUp() throws Exception
```

Comment:
Is this change directly relevant to this PR?

Reply:
I've moved this to PR #10180, once that's merged, this one will go away here.